docs/components/plugins/api/APIDataTypes.ts

```diff
@@ -148,7 +148,7 @@ export type TypeDeclarationContentData = {
 export type TypeDeclarationData = {
   name?: string;
   declaration?: TypeDeclarationContentData;
-  type: string;
+  type?: string;
```

Comment:
this appeared to be the correct way to resolve the merge conflict, given https://github.com/expo/expo/commit/a404f439750d056b6a4c56f984d619c341759610#diff-6fef0d20e1c51cb2883f31090408839b29385d49769538b5c819cf4ed5369520